package.json

```diff
@@ -157,7 +157,7 @@
         "requirejs-json": "0.0.3",
         "requirejs-plugins": "1.0.2",
         "select2": "4.0.13",
-        "select2-bootstrap-theme": "0.1.0-beta.9",
+        "select2-bootstrap-theme": "0.1.0-beta.10",
```

Comment:
Bumped this to the latest version. Incidentally, we're not actually installing it right during runtime, so it's not even in use. But that's a separate issue.

Reply:
😖